### PR TITLE
 Filter reorganization

### PR DESCRIPTION
**Pull request by stef74**
Move rating up
Move climbing type up

Need to test and add screenshoot

### CODE REVIEW

**Comment by stef74** [comment]
I have do some merge involuntary ... I'm not very mastering with using multiple branch. I don't know what I do to clean. Only 3 commits are related on this branch (filter.html)

**Comment by stef74** [comment]
see some screenshot : 1 by activity 
capture d ecran 2017-01-26 a 20 52 37

**Comment by fbunoz** [comment]
You didn't wait for the end of image uplod, then only one is displayed, the others are lost.

**Comment by stef74** [comment]
Grrrr we have to wait satruday evening

**Comment by stef74** [comment]
Only relevant commit : 
 	
reorg filter route 		
	6ec7dfc
 	move up climbing_outdoor_type 		
	b161666
 	down altitude info 		
	a7064a6

**Comment by brunobesson** [comment]
>Only relevant commit

Sorry, but that's not an option :confused:  Please clean up your branch, it's too hard trying to decypher or test your commits otherwise.

**Comment by stef74** [comment]
Yes I know but i don't know how to clean ( I can trash and create a proper branch ... only one file is changed)

**Comment by brunobesson** [comment]
>i don't know how to clean

@asaunier gave you some advice by email. I know it's a bit hard to master at first, but it's a real good investment in time. Rebasing is'nt that complicated.

**Comment by stef74** [comment]
capture d ecran 2017-01-26 a 20 52 37
capture d ecran 2017-01-26 a 20 52 19
capture d ecran 2017-01-26 a 20 52 13
capture d ecran 2017-01-26 a 20 51 49
capture d ecran 2017-01-26 a 20 51 36
capture d ecran 2017-01-26 a 20 51 18
capture d ecran 2017-01-26 a 20 51 06
capture d ecran 2017-01-26 a 20 51 01
capture d ecran 2017-01-26 a 20 50 53
capture d ecran 2017-01-26 a 20 50 47

**Comment by stef74** [comment]
@brunobesson @asaunier it's rebased.

File: c2corg_ui/templates/route/filters.html [hunk 1]
```diff
@@ -37,67 +37,8 @@
...
-                    filtersCtrl.filters.act.indexOf('mountain_biking') !== -1)">
-        <div class="col-xs-12 col-sm-6 filter">
-          <label translate>route_length</label><br>
-          <app-slider filter="rlen" filters-list="filtersCtrl.filters" max="100000" step="500" unit="m"></app-slider>
-        </div>
-      </div>
-
-      ## SKITOURING
+        
+        ## SKITOURING
```
Comment (brunobesson) [inline c2corg_ui/templates/route/filters.html]: Please do not add unnecessary indentation

File: c2corg_ui/templates/route/filters.html [hunk 2]
```diff
@@ -123,44 +64,51 @@
             values-list="['${"','".join(labande_ski_ratings) |n}']"></app-slider>
         </div>
       </div>
-
-      ## ROCK CLIMBING
+
```
Comment (brunobesson) [inline c2corg_ui/templates/route/filters.html]: ditto for indentation

**Comment by stef74** [comment]
Matthieu & Bubu have review the fonctionnal part (in forum). I will fix the identation issue

**Comment by asaunier** [comment]
@stef74 Pay attention to your commit messages. Those are made available in the commits history when the PR is merged. It is important that messages are clear and understandable out of the PR context. Anyway I recommend to squash the commits when merging because 5 commits are not really useful for this change.

Else OK for merging if you had good feedbacks from the association, thanks.

**Pull request merged**